process.h

```diff
@@ -143,4 +143,13 @@ void refreshconninode ();
 
 void procclean ();
 
+void getkbps (Process * curproc, float * recvd, float * sent);
```

Comment:
The problem with these getters...() functions is that they **modify** the Process structure and they are not simple getters !
the 3 conversion functions could be moved.
Let me see if I can't find a better way to move them.